.github/workflows/release_gh.yaml

```diff
@@ -167,18 +162,35 @@ jobs:
       - name: Create release
         run: |
           ls -R
-          gh release create "v$VERSION" \
-            --target "$GITHUB_REF_NAME" \
-            --notes '${{ steps.extract-release-notes.outputs.release_notes }}' \
-            "Capture-$VERSION.ios.zip" \
-            "Capture-$VERSION.doccarchive.ios.zip" \
-            "Capture-$VERSION.android.zip" \
-            "example-apps.ios.zip" \
-            "example-apps.android.zip" \
-            "capture-timber-$VERSION.android.zip" \
-            "capture-apollo-$VERSION.android.zip" \
-            "capture-plugin-$VERSION.android.zip" \
+          RELEASE_NOTES='${{ steps.extract-release-notes.outputs.release_notes }}'
```

Comment:
kinda wonder if this is gonna be a problem later with escaping, but I'm sure we'll find out!